hail/src/main/scala/is/hail/expr/types/physical/PVoid.scala

```diff
@@ -13,22 +13,5 @@ case object PVoid extends PType {
 
   override def _pretty(sb: StringBuilder, indent: Int, compact: Boolean): Unit = sb.append("PVoid")
 
-  def codeOrdering(mb: EmitMethodBuilder[_], other: PType): CodeOrdering = null
-
-  def copyFromType(mb: EmitMethodBuilder[_], region: Value[Region], srcPType: PType, srcAddress: Code[Long], deepCopy: Boolean) =
-    throw new UnsupportedOperationException("PVoid copyFromType is currently undefined")
-
-  def _copyFromAddress(region: Region, srcPType: PType, srcAddress: Long, deepCopy: Boolean) =
-    throw new UnsupportedOperationException("PVoid copyFromType is currently undefined")
-
-  def copyFromTypeAndStackValue(mb: EmitMethodBuilder[_], region: Value[Region], srcPType: PType, stackValue: Code[_], deepCopy: Boolean): Code[_] =
```

Comment:
I think this is a bug. need to implement this

Reply:
Can you explain why? What is a stack value for `PVoid`?

Reply:
I think it's as much a problem with the usages, but we currently do `copyFomTypeAndStackValue` in the aggregators to unify seqOpArgs -- some of these arguments are void.

It's a way to avoid checks for void everywhere. This code will change soon, but we need it right now.

Reply:
The version in this PR inherits its implementation from `PUnrealizable`, which throws. I can make `PVoid` override this with its old implementation, but this doesn't seem to be causing any test failures, and "find usages" isn't finding the use in aggregators.

If you're talking about the emit rule for `SeqOp`, that uses `copyFromPValue`, which is defined in `PUnrealizable` to have the same behavior as the old `PVoid.copyFromPValue`. This doesn't have the same problem as `copyFromTypeAndStackValue`, because you're "copying" a `PUnrealizableCode`, which just returns the pcode, not copying a stack value, which doesn't make sense for unrealizable types.